### PR TITLE
MW-45 Cta article page

### PR DESCRIPTION
**Pull request by padsbanger**
- CTA in the middle of article
- CTA on the right sidebar
- CTA on the bottom of the page
- All CTA sections are different between US and PL version

https://masterborn.atlassian.net/browse/MW-45

### CODE REVIEW

File: src/components/SidebarCta.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  margin: 1rem 0;
+  background:  ${props => props.theme.colors.primary};
+`
+
+const SidebarCta = ({ isInPoland }) => {
+  return (
+    <Container>
+      <StyledHeading as="h5">We build valuable products</StyledHeading>
+      <StyledBorder />
+      <Button
```
Comment (jBosak98) [inline src/components/SidebarCta.js:40]: We probably need some kind of redirect to '/contact' here.

File: src/components/mdxComponents/CtaArticleComponent.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+const CtaArticleComponent = () => {
+  const { isInPoland } = useContext(CountryContext);
+  return (
+    <Container>
+      <StyledHeading as="h5" mb={0}>{
+        isInPoland ? "Let’s build disruptive JavaScript products together"
+          : 'Build your modern Web App with top React & Node.js Engineers'}
+      </StyledHeading>
+      <StyledButton
```
Comment (jBosak98) [inline src/components/mdxComponents/CtaArticleComponent.js:45]: same as above - redirect

**Review by jBosak98 — COMMENTED** [review]
well done :)

File: src/components/SidebarCta.js (hunk 1 above)
Comment (padsbanger) [reply src/components/SidebarCta.js:40]: Thats part of another user story. On Polish version we redirect to `/contact` on English we show a modal with form, but currently we dont have design for modal.

**Review by jBosak98 — APPROVED** [review]

**Pull request merged**